src/Dapr.Actors/Client/ActorProxy.cs

```diff
@@ -45,6 +45,7 @@ protected internal ActorProxy()
 
         internal IActorMessageBodyFactory ActorMessageBodyFactory { get; set; }
         internal JsonSerializerOptions JsonSerializerOptions = JsonSerializerDefaults.Web;
+        internal string DaprApiToken;
```

Comment:
nit: `daprApiToken`

Reply:
If this needs to be internal, it would be great to make this a property instead so it's not settable outside the class.